src/main/client/app/shared/components/markdown.vue

```diff
@@ -0,0 +1,30 @@
+<template>
+  <div class="markdown-body">
+    <!-- eslint-disable-next-line vue/no-v-html -->
+    <div v-html="convertedContent" />
```

Comment:
`<div>{{ convertedContent }}</div>`

Reply:
I don't think it will work. We need to display HTML.

https://vuejs.org/v2/guide/syntax.html#Raw-HTML